.changeset/hot-onions-impress.md

```diff
@@ -0,0 +1,6 @@
+---
+'@chainlink/proof-of-reserves-adapter': minor
+'@chainlink/cryptoapis-adapter': minor
+---
+
+Add cryptoapis integration tests
```

Comment:
Integration tests isn't part of the source files that are changed, the real change here is that the way exports are done has been changed.

I also don't think we need to include `@chainlink/proof-of-reserves-adapter` explicitly here. @justinkaseman what do you think?

Reply:
Cryptoapis src export change

Reply:
```suggestion
Cryptoapis src export change
```